app/models/main.py

```diff
@@ -771,9 +774,13 @@ def validate_email_address(self, key, value):
     @validates('role')
```

Comment:
So in the method above we do the role check for buyers on email change too if we're changing an email address. do we want to do this for admins too? (Should we even be doing it for buyers?)

Reply:
Oops I didn't even see that one. I think checking both is a good idea - will fix.